### ISSUE TITLE
Cannot access files

### ISSUE THREAD

**Issue by geoo89** [IDEMSInternational/r-instat-docker#1]
When trying to open a file in R-Instat, the only files that I can access are those on my desktop.
I don't seem to be able to access anything else in my file system when traversing the tree structure that's offered in the file dialog (see below). 

![image](https://user-images.githubusercontent.com/678199/154226333-663a60fd-f08d-4050-aafa-5dedce03c52a.png)

For the record, these are my partitions:
![image](https://user-images.githubusercontent.com/678199/154226519-19d56ec9-2343-4798-8997-432aeeaf30be.png)

I'm running R-Instat from the sda3 partition.

**Comment by dannyparsons** [comment]
@geoo89 Yes, this is a current limitation and something that would be great to improve on.

I believe Ian tried mounting the Home directory but ran into issues on this. @istride could probably explain the issue with this.

If there were any simple improvements for this that would be great.

**Comment by istride** [comment]
One of the disadvantages of using docker to run R-Instat is that you have to explicitly mount volumes on the container from the host system. To do this you would add another entry to the list of volumes in the docker-compose.yml file. The format is `<host system path>:<container path>`.
```
volumes:
      - /tmp/.X11-unix:/tmp/.X11-unix
      - ~/.Xauthority:/home/rinstat/.Xauthority
      - ~/Desktop:/home/rinstat/Desktop
      - /extra/volume:/data
```
As you can see, the Desktop directory in the user's home directory is already mounted in the container, so this could be used as a staging area for data into and out of R-Instat too.

Mounting an entire user home directory from the host into the container interferes with the shutdown process of R-Instat, at the very least, so I would avoid doing that.
